src/utils/gaa.js

```diff
@@ -970,6 +970,10 @@ export class GaaSignInWithGoogleButton {
           'theme': 'outline',
           'text': 'continue_with',
           'logo_alignment': 'center',
+          'width': self.document.getElementById(SIGN_IN_WITH_GOOGLE_BUTTON_ID)
```

Comment:
Could you create a local variable for this button element? (`self.document.getElementById(SIGN_IN_WITH_GOOGLE_BUTTON_ID)`). That would allow the script to just query for it once, and could improve readability too

Reply:
Oh you could actually just reference `buttonEl` directly

Reply:
Done.